README.md

```diff
@@ -21,6 +25,44 @@ You may have noticed that specifying "production" created an artifact from the m
 
 Before the artifact is built Jarå will check that _branch_name_ and origin/*branch_name* point to the same commit. The reason for this is so that you don't release an artifact with a SHA that is not visible to others (this does not check that you've pulled before you release, but the important thing is to not release something that is not trackable).
 
+The same can be accomplished by running this from the command line:
+
+```
+$ jara --environment production --release --bucket artifact-bucket
+```
```

Comment:
Having a `build` and a `release` subcommand sounds like a good idea.

`jarå` is just asking for trouble.